.babelrc

```diff
@@ -1,11 +1,20 @@
 {
 	"presets": [
-		"es2015",
+		[ "env", {
+			"targets": {
+				"browsers": [
+					"last 2 versions",
+					"Safari >= 9",
```

Comment:
I know this one! :)

Safari versions (at least for iOS), correspond with `iOS` releases, so you have these:
* Safari 8
* Safari 8.1
* Safari 9
* Safari 9.1

With the `last 2 versions` rule, after `iOS 9.1` was released, Calypso broke on `iOS 8.x`. Sadly, there's no way to tell `browserslist` `last 2 MAJOR versions`.